www/cordova/apphost.js

```diff
@@ -205,9 +205,9 @@ define(['appStorage', 'browser'], function (appStorage, browser) {
             features.push('externallinks');
             features.push('multiserver');
 
-            if (MainActivity.getChromeVersion() >= 53) {
-                features.push('imageanalysis');
-            }
+            //if (MainActivity.getChromeVersion() >= 53) {
```

Comment:
Since this method was removed from MainActivity.java, I think this should be removed completely.

Reply:
I commented these three sections out just in case we need to reference them while we are migrating the cordova stuff into a plugin, but if you want to remove them now I can make the change.

Reply:
I don't see a value in leaving references in the JavaScript to functionality that has been removed on the Java side. If something comes up we could always reference the git history.

I don't know that these will need to be added to a plugin since the values were hardcoded and the email functionality only seemed to be used if there was an issue with in app purchases.